index.html

```diff
@@ -0,0 +1,296 @@
+            <h2 class="html">css</h2>
+          </li>
+          <li class="java">
+            <h2 class="html">javaScript</h2>
+          </li>
+        </ul>
+        <div id="action">
+          <button type="button" class="normalbuttn"> See project</button>
+        </div>
+      </div>
+    </div>
+  </section> 
+  
+  <section class="about-myself-section">
+
+    <div class="abouttop">
+      <h1 class="aboutheadertext">
+        About  Myself
+      </h1>
+      <p 
+        class="aboutbodytext">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="socialmedia">
+        <h4 class="supportingText">
+          LET'S CONNECT
+        </h4>
+        <div class="frame2">
+          <ul class="icons">
+            <li><a href="twitter.com/Mc_Emarkees" class="fa-brands fa-twitter"></a></li>
+            <li><a href="github.com/emarkees" class="fa-brands fa-github"></a></li>
+            <li><a href="linkedin.com" class="fa-brands fa-linkedin-in"></a></li>
+            <li><a href="medium.com" class="fa-brands fa-medium"></a></li>
+            <li><a href="angellist.com" class="fa-brands fa-angellist"></a></li>
+          </ul>
+        </div>
+      </div>
+      <div id="action">
+        <button class="normalbutnaboutt"> Get my resume</button>
+      </div>
+      
+    </div>
+
+    <div class="skillist">
+      <div class="dropdowncontainer">
+        <div id="dropdownheader">
+          <h3 class="droptext">Languages</h3>
+          <i class="dropdownhamburger">
+            <i class="fa-solid fa-angle-down"></i>
+          </i>
+        </div>
+        <div class="dropdown">
+          <div id="javadropdown">
+            <img src="./asset/About/javaScript.svg" alt="" class="image">
+            <h4 class="droptxt">Java</h4>
+          </div>
+          <div id="javadropdown">
+            <img src="./asset/About/Html.svg" alt="" class="image">
+            <h4 class="droptxt">HTML</h4>
+          </div>
+          <div id="javadropdown">
+            <img src="./asset/About/Css.svg" alt="" class="image">
+            <h4 class="droptxt">CSS</h4>
+          </div>        
+        </div>         
+      </div>
+      
+      <div class="dropdowncontainer">
+        <div id="dropdownheader">
+          <h3 class="droptext">Frameworks</h3>
+          <i class="dropdownhamburger">
+            <i class="fa-solid fa-angle-right"></i>
+          </i>
+        </div>
+      </div>
+      <hr class="hr"></hr>
+      <div class="dropdowncontainer">
+        <div id="dropdownheader">
+          <h3 class="droptext">Skills</h3>
+          <i class="dropdownhamburger">
+            <i class="fa-solid fa-angle-right"></i>
+          </i>
+        </div>
+      </div>
+      <hr class="hr"></hr>
+      
+    </div>
+  </section>
```

Comment:
- [ ] In my opinion it would be better to make sure design is not breaking in order to stick to the Figma design.
Your design:
![Screenshot from 2023-03-04 14-25-03](https://user-images.githubusercontent.com/97834160/222901067-f69ee119-0db7-419d-b776-9972656ce8fc.png)
Figma:
![Screenshot from 2023-03-04 14-30-01](https://user-images.githubusercontent.com/97834160/222901126-c1ccb36c-e690-454c-9143-8a9d92f6dc11.png)

Reply:
Hello Sir,
issue resolved the design was made as per the Microverse desktop version (Desktop: >=768px).one in the Figma design template.